packages/calypso-products/src/plans-list.tsx

```diff
@@ -1413,6 +1413,7 @@ const getPlanPremiumDetails = (): IncompleteWPcomPlan => ( {
 		FEATURE_PREMIUM_THEMES_V2,
 		FEATURE_UNLTD_SOCIAL_MEDIA_JP,
 		FEATURE_VIDEOPRESS_JP,
+		FEATURE_STATS_PAID,
```

Comment:
This is the Newsletter flow.
<img width="1438" alt="newsletter" src="https://github.com/Automattic/wp-calypso/assets/140841/bea7d753-498e-4887-ab07-a731cc64a50f">